src/components/CountdownTimer.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from "react";
```

Comment:
:wrench: This needs to be in its own directory.

Reply:
Addressed:  0141e30395a7c95559f408514669b8dae7d1439f